runtime/src/metrics/histogram.rs

```diff
@@ -0,0 +1,47 @@
+use prometheus_client::metrics::histogram::Histogram;
+use std::time::SystemTime;
+
+/// Holds constants for bucket sizes for histograms.
+///
+/// The bucket sizes are in seconds.
+pub struct Buckets;
+
+impl Buckets {
+    /// For roundtrip requests to a peer.
+    pub const P2P: [f64; 12] = [
```

Comment:
I think we should remove `P2P` here and juse use `NETWORK`

Reply:
I can see the reasoning but I felt they are a bit different. For example I think the "time to come to consensus/threshold" (network) is usually meaningfully different than "time to get a ping/pong response" (p2p). Although maybe it's fair that they are not off by enough orders of magnitude to matter

Reply:
I think we should be relying on the metrics themselves to delineate meaningfully between the two.

I think the values in both should be very similar (within ~2-3x) and we should reuse to avoid confusion.